Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/sdk:latest as build
```

Comment:
since we are working on dotnet 8.0, we should specify it in case of compatibility issues with future dotnet releases. same for the aspnet below